src/main/java/org/broadinstitute/consent/http/db/MatchDAO.java

```diff
@@ -12,53 +12,54 @@
+    @SqlQuery("SELECT * FROM match_entity WHERE purpose = :purposeId ")
+    List<Match> findMatchesByPurposeId(@Bind("purposeId") String purposeId);
 
-    @SqlQuery("select * from match_entity where purpose = :purposeId and consent = :consentId ")
-    Match  findMatchByPurposeIdAndConsent(@Bind("purposeId") String purposeId, @Bind("consentId") String consentId);
+    @SqlQuery("SELECT * FROM match_entity WHERE purpose = :purposeId AND consent = :consentId ")
+    Match findMatchByPurposeIdAndConsentId(@Bind("purposeId") String purposeId, @Bind("consentId") String consentId);
 
-    @SqlQuery("select * from match_entity where matchId = :id ")
+    @SqlQuery("SELECT * FROM match_entity WHERE matchid = :id ")
     Match  findMatchById(@Bind("id") Integer id);
 
-    @SqlQuery("select * from match_entity where purpose  IN (<purposeId>)")
-    List<Match>  findMatchesPurposeId(@BindList("purposeId") List<String> purposeId);
+    @SqlQuery("SELECT * FROM match_entity WHERE purpose IN (<purposeId>)")
+    List<Match> findMatchesForPurposeIds(@BindList("purposeId") List<String> purposeId);
 
-    @SqlUpdate("insert into match_entity " +
-            "(consent, purpose, matchEntity, failed, date) values " +
-            "(:consentId, :purposeId, :match, :failed, :createDate)")
+    @SqlUpdate(
+            " INSERT INTO match_entity " +
+            " (consent, purpose, matchentity, failed, createdate) VALUES " +
+            " (:consentId, :purposeId, :match, :failed, :createDate)")
     @GetGeneratedKeys
     Integer insertMatch(@Bind("consentId") String consentId,
                         @Bind("purposeId") String purposeId,
                         @Bind("match") Boolean match,
                         @Bind("failed") Boolean failed,
                         @Bind("createDate") Date date);
 
-    @SqlBatch("insert into match_entity (consent, purpose, matchEntity, failed, createDate) values (:consent, :purpose, :match, :failed, :createDate)")
+    @SqlBatch("INSERT INTO match_entity (consent, purpose, matchentity, failed, createdate) VALUES (:consent, :purpose, :match, :failed, :createDate)")
     void insertAll(@BindBean List<Match> matches);
 
-    @SqlUpdate("update match_entity set matchEntity = :match, consent = :consentId, purpose = :purposeId, failed = :failed where matchId = :id ")
-    void updateMatch(@BindList("match") Boolean match,
+    @SqlUpdate("UPDATE match_entity SET matchentity = :match, consent = :consentId, purpose = :purposeId, failed = :failed WHERE matchid = :id ")
+    void updateMatch(@Bind("id") Integer id,
+                     @Bind("match") Boolean match,
                      @Bind("consentId") String consent,
                      @Bind("purposeId") String purpose,
                      @Bind("failed") Boolean failed);
 
-    @SqlBatch("DELETE FROM match_entity WHERE matchid IN (<matchIds>)")
-    void deleteMatches(@BindList("matchIds") Collection<Integer> matchIds);
```

Comment:
This was throwing strange type errors and was also unnecessary as we can delete by other means more efficiently.